packages/valaxy/node/plugins/markdown/transform/index.ts

```diff
@@ -2,6 +2,9 @@ import type { Plugin } from 'vite'
 import Markdown from 'unplugin-vue-markdown/vite'
 
 import type MarkdownIt from 'markdown-it'
+import matterOptions from 'packages/valaxy/node/utils/matterOptions'
```

Comment:
Hey, let's use relative path to avoid incorrect path resolution.
And use on demand import `import { matterOptions } from 'xxx'` instead of import default.